src/modules/commander/commander_params.c

```diff
@@ -960,19 +960,18 @@ PARAM_DEFINE_INT32(COM_ARMABLE, 1);
 /**
  * Minimum battery level for arming
  *
- * Additional battery level check that only allows arming if the state of charge of the emptiest
- *  connected battery is above this value.
+ * Threshold for battery percentage below arming is prohibited.
  *
- * A value of 0 disables the check.
+ * A negative value means BAT_CRIT_THR is the threshold.
  *
  * @unit norm
- * @min 0
+ * @min -1
  * @max 0.9
  * @decimal 2
  * @increment 0.01
  * @group Commander
  */
-PARAM_DEFINE_FLOAT(COM_ARM_BAT_MIN, 0.f);
+PARAM_DEFINE_FLOAT(COM_ARM_BAT_MIN, -.01f);
```

Comment:
We usually use -1 in these cases (to indicate that something is turned off). I would stay consistent here.

Reply:
I agree, I actually did that initially but then changed because -100% looked odd in the ground station:
![grafik](https://github.com/user-attachments/assets/ec915619-ac8e-4556-a8bc-3dfa96aa0942)
Let's stay consistent, I'll change it back 👍

Reply:
Changed in place: https://github.com/PX4/PX4-Autopilot/compare/1eb2bec7cc5721b706e7e73341649e0f3eab6d2c..ea5641fa7f4134318da29e6a5a9ad59480671266